pyubee/__init__.py

```diff
@@ -62,6 +149,24 @@
             r'<td>([0-9a-fA-F]{12})</td>'  # mac address
             r'<td>(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})</td>'  # ip address
         ),
+        'authenticator': DefaultAuthenticator
+    },
+    'EVW3226@UPC': {
```

Comment:
Autodiscovery does not work for it (no model data without logging in), so I chose this name, which has to be typed manually as script argument.

Reply:
Does your device not have a /RootDevice.xml file? Perhaps you need to enable UPnP support on your device first?

Reply:
It does not have it and I have UPnP enabled. I appears that the firmware was heavily modified by UPC. Router pages available without logging in do no contain such information.